### PR TITLE
Add blank component

### PR DESCRIPTION
**Pull request by Cuel**
Adds the mentioned blank component folder

### CODE REVIEW

**Comment by Cuel** [comment]
tagging @jonpas

File: blank/script_component.hpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#define COMPONENT blank
+#define COMPONENT_BEAUTIFIED Blank
+#include "z\proj_templ\addons\main\script_mod.hpp"
+
+// #define DEBUG_MODE_FULL
+// #define DISABLE_COMPILE_CACHE
+// #define CBA_DEBUG_SYNCHRONOUS
```
Comment (jonpas) [inline blank/script_component.hpp]: Not necessary since ACE3 defines synchronous debugging in main macros file.

**Comment by Glowbal** [comment]
Lets place it in the extras folder. Thanks @Cuel for this. :+1:

**Comment by Cuel** [comment]
Moved into extras folder and update `CfgPatches` in main

**Pull request merged**